css/about.css

```diff
@@ -0,0 +1,167 @@
+
+#hero h2 {
+  margin-left: 50px;
+}
+
+#hero h3 {
+  margin: 5px 50px;
+  color: #ec5242;
+  text-overflow: inherit;
+}
+
+.date-place h4 {
+  font-size: 2rem;
+  margin-left: 20px;
+}
+
+.date-place a {
+  color: black;
+  border-bottom: 1px solid black;
+}
+
+#ourlogo {
+  height: 60vh;
+  padding: 20px;
+}
+
+#ourlogo .container {
+  justify-content: center;
+  align-items: center;
+}
+
+#ourlogo .container h4 {
+  margin: 10px 0;
+  font-weight: normal;
+}
+
+#ourlogo .container img {
+  height: 25vh;
+  width: 60vw;
+  border: 3px solid #ec5242;
+  padding: 5px;
+  margin-top: 5px;
+}
+
+.last-activities {
+  background-color: #272a31;
+  color: white;
+  height: 80vh;
+  text-align: center;
+  font-weight: 900;
+}
```

Comment:
- [ ] The background and font colors should comply with the original design.
![image](https://user-images.githubusercontent.com/64894927/128205234-97485136-6308-49ec-9200-c08c7674b467.png)
![image](https://user-images.githubusercontent.com/64894927/128205308-7e3d82c3-f892-412d-a275-e5356bce3087.png)

Reply:
Done thanks for your review 👍